### DISCUSSION TITLE
Meaning of opt = true

### DISCUSSION THREAD

**Discussion by jtbx** [wbthomason/packer.nvim#1146] [Q&A] [answered]
What does `opt = true` mean inside a `requires` table? Does it just stand for optional?

**Comment by lewis6991** [comment, marked as answer]
Plugins can either be `start` or `opt`:
- `start` plugins are loaded automatically after `init.lua`
- `opt` plugins must be manually loaded via `:packadd <plugin_name>`

See `:help packages`

**Reply by jtbx** [reply]
Thanks!

**Reply by nyngwang** [reply]
Why people will want to manually load something during development?
